### DISCUSSION TITLE
Handle abandoned carts in admin

### DISCUSSION THREAD

**Discussion by Hemann55** [medusajs/medusa#2896] [Feature Requests]
It would be great if abandoned carts were visible in Admin along with actions like sending email reminders and offering discount. These actions can be manual or automated depending on store owner's preference

**Comment by herawais** [comment]
any update so far ?
otherwise i will be working on that functionality and create a PR

**Comment by olivermrbl** [comment]
This is an excellent idea. Very useful to many. However, it is highly unlikely we'll work on such a feature soon, as there are many other things in our backlog with higher priority.

I suggest you build this functionality in your own project or as a plugin, so you are not held back by our PR review process. You can use our [UI Routes](https://docs.medusajs.com/admin/routes) to add a separate tab in our sidebar specifically for abandoned carts. This approach gives you the freedom to design the actions so that they fit your exact use case. 

And you can use our [UI Library](https://docs.medusajs.com/ui) for components to build a design-consistent experience.

**Reply by herawais** [reply]
Thanks for your reply, i will start building the plugin and publish it on npm too

**Reply by luluhoc** [reply]
https://github.com/luluhoc/medusa-plugin-abandoned-cart

**Comment by Kainkainkain** [comment]
Hi @herawais - did you have success with an abandoned cart feature?

**Reply by herawais** [reply]
@Kainkainkain Yes i am able to develop the plugin
and it is working fine for my client

**Reply by Kainkainkain** [reply]
Oh that's awesome - Is it publicly available on npm? I would love to use such a feature- I've been looking at having external Cron job run a lambda to discover and then process and a solution within Medusa would be so much better looking and manageable

**Reply by mdornellas** [reply]
Great! Would also love to use this :D

**Reply by Gustavodacrvi** [reply]
I'm looking for this feature as well

**Reply by SautronAndCo** [reply]
Hello @herawais would you mind sharing this plugin please ?

**Reply by luluhoc** [reply]
@olivier97431 https://www.npmjs.com/package/medusa-plugin-abandoned-cart

**Comment by luluhoc** [comment]
I wrote a plugin

https://github.com/luluhoc/medusa-plugin-abandoned-cart

https://www.npmjs.com/package/medusa-plugin-abandoned-cart

It has widget admin as well, it is only manual send for now, and will add some new feats soon feel free to open PR

**Reply by luluhoc** [reply]
The plugin received massive update with schedule and more
